Dockerfile

```
@@ -0,0 +1,49 @@
+RUN cargo build --release --package onshape-mcp 2>/dev/null || true
+
+# Copy full source and build for real
+COPY crates/ crates/
+RUN touch crates/*/src/*.rs crates/*/src/**/*.rs 2>/dev/null; \
+    cargo build --release --package onshape-mcp
+
+# Stage 2: Minimal runtime image
+FROM debian:bookworm-slim
+
+# Install CA certificates (needed for HTTPS to Onshape API)
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)